x/bank/types/balance.go

```diff
@@ -64,9 +64,14 @@ func SanitizeGenesisBalances(balances []Balance) []Balance {
 
 	// 1. Retrieve the address equivalents for each Balance's address.
 	addresses := make([]sdk.AccAddress, len(balances))
+	existingAddresses := map[string]bool{}
 	for i := range balances {
 		addr, _ := sdk.AccAddressFromBech32(balances[i].Address)
 		addresses[i] = addr
+		if _, exists := existingAddresses[string(addr)]; exists {
+			panic("duplicate account in genesis state")
```

Comment:
Please add more context to the panic otherwise users will scratch their heads then have to go comb through all the balances' addresses: please do
```go
if _, ok := seen[string(add)]; ok {
   panic(fmt.Sprintf("Genesis state has duplicate account: %q aka %x", addr, addr))
```